src/components/my-component/my-component.tsx

```diff
@@ -325,9 +433,12 @@ export class MyComponent {
                     {
                         source: item.pid,
                         target: propValue,
-                        type: propKey
+                        type: propKey,
+                        category: 'non_attribute'
                     }
-                    if (this.showPrimaryLinks) links.push(link);
+                    if (this.showPrimaryLinks) { conditionalLinks.push(link); }
```

Comment:
Apart from the naming as 'conditionalLinks' (what does this mean) this handling looks strange. If showPrimaryLinks is true, the link is put into conditionalLinks. If you look at line 466 it reads:

`     links = this.showPrimaryLinks ? linkList : conditionalLinks;`

Here, if showPrimaryLinks is true, conditionalLinks are ignored.

I would suggest to: 

1) Rename variables to something more readable/understandable
2) Check the code for logical errors
3) Think about only using one list and add a property 'visible' or something, so you don't have to maintain two link lists.